modules/las/src/lib/parse-las.ts

```diff
@@ -62,7 +76,7 @@ export default function parseLAS(arrayBuffer: ArrayBuffer, options?: LASLoaderOp
       positions[pointIndex * 3 + 1] = position[1] * scaleY + offsetY;
       positions[pointIndex * 3 + 2] = position[2] * scaleZ + offsetZ;
 
-      if (color) {
+      if (color && colors !== null) {
```

Comment:
```suggestion
      if (color && colors) {
```

Reply:
@ibgreen  Ok, fixed everything in https://github.com/visgl/loaders.gl/pull/1603/commits/26f67c6e2949cd89b8c73f9807d1503604fe29ca